pcp/Platform.c

```diff
@@ -0,0 +1,874 @@
+/*
+htop - linux/Platform.c
+(C) 2014 Hisham H. Muhammad
+(C) 2020-2021 htop dev team
+(C) 2020-2021 Red Hat, Inc.  All Rights Reserved.
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h"
+
+#include <math.h>
+
+#include "BatteryMeter.h"
+#include "ClockMeter.h"
+#include "Compat.h"
+#include "CPUMeter.h"
+#include "DateMeter.h"
+#include "DateTimeMeter.h"
+#include "DiskIOMeter.h"
+#include "HostnameMeter.h"
+#include "LoadAverageMeter.h"
+#include "Macros.h"
+#include "MainPanel.h"
+#include "MemoryMeter.h"
+#include "Meter.h"
+#include "NetworkIOMeter.h"
+#include "Object.h"
+#include "Panel.h"
+#include "PCPProcess.h"
+#include "PCPProcessList.h"
+#include "Platform.h"
```

Comment:
Prefer to have relative paths based on main directory, thus including the platform directory.

Reply:
Hmm?  That's not what other platforms seem to do... e.g. grep ProcessList.h darwin/*c linux/*c

Reply:
Fixing this in next update - thanks @BenBE